python/lsst/pex/config/configurableActions/_configurableAction.py

```diff
@@ -30,8 +30,8 @@
 
 
 class ConfigurableAction(Config):
-    """A `ConfigurableAction` is an interface that extends a
-    `lsst.pex.config.Config` class to include a `__call__` method.
+    """A `~.configurableActions.ConfigurableAction` is an interface that
```

Comment:
This is another one that really looks like it should have worked previously (lots more like it in this commit).

Reply:
I think this is another place where lsst.pex.config.configurableActions being its own package gets us in to trouble. Without these changes it couldn't work out what `ConfigurableAction` was.

Reply:
I agree this is analogous to the other case - here configurable actions are in play, but we're in that subpackage, so it should still work.

Reply:
This one you are correct. I think I know what happened here. Before I made it a sphinx module of its own sphinx couldn't find any of these classes because they were not lifted to `lsst.pex.config`. I confirm that removing these prefixes works.